examples/minimal.js

```diff
@@ -17,7 +17,10 @@ const serverStream = socket1
 const clientStream = socket2
 
 const { close } = createServer(myApi, serverStream)
-const myApiOnClient = createClient(clientStream)
+
+const myApiOnClient = /** @type {import('..').ClientApi<typeof myApi>} */ (
```

Comment:
Are we exporting types so we could do?
```js
const clientApi = /** @type {import('rpc-reflector).ClientApi<typeof serverApi> */
```
or could we just do?
```js
const clientApi = /** @type {typeof serverApi} */
```

Reply:
Yes, this now exports the type `ClientApi` which you can use to get the client API from your server API (the type is different because all sync methods are converted to async).

If you are writing Typescript you can just do:

```ts
const clientApi = createClient<typeof serverApi>(port)
```

However when writing js with Typescript as JSDoc there is no easy way to call a generic, so you need to do:

```js
const clientApi = /** @type {import('rpc-reflector).ClientApi<typeof serverApi> */ (createClient(port))
```

Note the brackets around `(createClient(port)` which is necessary to type cast (https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#casts)